python/pyspark/context.py

```diff
@@ -292,6 +292,20 @@ def version(self):
         return self._jsc.version()
 
     @property
+    def applicationId(self):
+        """
+        applicationId - unique identified for the Spark Application
```

Comment:
I would try to make this docstring match the style of our other docstrings a bit better.  Specifically, there's not a need to say `applicationId - ` here, since that's clear from the method's name.

Instead, I'd have this first line just read  something like this:

```
A unique identifier for the Spark application.
Its format depends on the scheduler implementation.
```

I'm not sure whether we need to show the examples here, but I guess it doesn't necessarily hurt.